wpilibcExamples/src/main/cpp/examples/Frisbeebot/include/Constants.h

```diff
@@ -62,7 +63,7 @@ inline constexpr double kD = 0;
 inline constexpr auto kS = 0.05_V;
 inline constexpr auto kV =
     // Should have value 12V at free speed...
-    12_V / kShooterFreeRPS;
+    12_V / units::radians_per_second_t{kShooterFreeRPS};
```

Comment:
kShooterFreeRPS should be a unit type instead of being wrapped here. This is especially a good idea since this line and the one in ShooterSubsystem.cpp give the same double different units.

Reply:
and done